migrations/20200719172434-change-patients-barcodeValue-to-string.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.removeColumn("patients", "barcodeValue", {
+          transaction: t,
+        }),
+        queryInterface.addColumn(
+          "patients",
+          "barcodeValue",
+          {
+            type: Sequelize.DataTypes.STRING,
+          },
+          { transaction: t }
+        ),
+      ]);
+    });
+  },
+
+  down: (queryInterface, Sequelize) => {
```

Comment:
similar issue here, except converting from a string to an int is not a clean operation - to keep things simple, i would only convert the string values that consist of only numbers (i.e. all "string" integers)

if you're having trouble with this, just ping me on slack

Reply:
fixed in latest commit, but not sure if it's correct